subworkflows/nf-scil/registration/main.nf

```diff
@@ -1,5 +1,8 @@
 include { REGISTRATION_ANATTODWI  } from '../../../modules/nf-scil/registration/anattodwi/main'
 include { REGISTRATION_ANTS   } from '../../../modules/nf-scil/registration/ants/main'
+include { REGISTRATION_EASYREG } from '../../../modules/nf-scil/registration/easyreg/main'
+
+params.run_surgery = params.run_surgery?: false
```

Comment:
Nextflow config inheritance is a feature of the language. You don't have to check for pre-initialization of the parameter here. If it has been defined in an higher level (workflow, pipeline, main config), it will override this value. Just set :

```suggestion
params.run_surgery = false
```